tests/foreman/ui/test_computeresource_vmware.py

```diff
@@ -0,0 +1,368 @@
+"""Test for compute resource UI
+
+@Requirement: Computeresource
+
+@CaseLevel: Acceptance
+
+@CaseComponent: UI
+
+@TestType: Functional
+
+@CaseImportance: High
+
+@Upstream: No
+"""
+
+from robottelo.decorators import run_only_on, stubbed, tier1, tier2, tier3
+from robottelo.test import UITestCase
+
+
+class VmwareComputeResourceTestCase(UITestCase):
+    """Implements compute resource tests in UI"""
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_positive_create_vmware_with_name(self):
+        """Create a new vmware compute resource using different data values
+        as name
+
+        @id: 944ed0da-49d4-4c14-8884-9184d2aef126
+
+        @setup: vmware hostname and credentials.
+
+        @steps:
+        1. Create a compute resource of type vmware.
```

Comment:
Add a blank line after @steps otherwise docs formatting won't be proper. Please run `make docs-clean docs` to generate docs and see if formatting is correct.